app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -118,19 +123,62 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
   const handleError = () => {
     handleChange(fundingAgreement.newFormData, "staged");
   };
+  const [showDiscardConfirmation, setShowDiscardConfirmation] = useState(false);
 
+  const handleDiscard = () => {
+    setShowDiscardConfirmation(false);
+    discardFundingParameterFormChange({
+      variables: {
+        input: {
+          revisionId: projectRevision.rowId,
+        },
+        connections: [],
+        reportType: "funding_parameter",
+      },
+    });
+  };
   return (
     <>
       {projectRevision.projectFundingAgreementFormChanges.edges.length ===
         0 && (
-        <Button onClick={addFundingAgreement} style={{ marginRight: "1rem" }}>
-          Add Funding Agreement
-        </Button>
+        <div>
+          <h3>Is this a funded project?</h3>
+          <RadioButton
+            name="yes"
+            label="Yes"
+            className="radio-button"
+            onClick={addFundingAgreement}
```

Comment:
This is a double-click bug, we should set a `disabled` field that depends on the `isAddingFundingParameterFormChange` property